site/content/en/docs/Installation/Creating Cluster/minikube.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Minikube"
+linkTitle: "Minikube"
+weight: 100
+description: >
+  This will setup a [Minikube](https://github.com/kubernetes/minikube) cluster, running on an `agones` profile.
+---
+
+
+
+## Installing Minikube
+
+First, [install Minikube][minikube], which may also require you to install
+a virtualisation solution, such as [VirtualBox][vb] as well.
+
+[minikube]: https://github.com/kubernetes/minikube#installation
```

Comment:
Maybe point to https://minikube.sigs.k8s.io/docs/start/ instead?

Reply:
I hadn't seen this before! Making the change. :+1: